pkg/handler/v2/backupcredentials/backupcredentials.go

```diff
@@ -70,6 +71,33 @@ func CreateOrUpdateEndpoint(userInfoGetter provider.UserInfoGetter) endpoint.End
 			}
 		}
 
+		// if destination is set, means we are using the new multiple backup destinations
```

Comment:
can we create a new provider with the specific functionality instead of exposing the master client directly to the endpoint

Reply:
or use BackupCredentialsProvider?

Reply:
We need a SeedProvider, also in the Seed endpoints(they also use master client directly). I wanted to do it in a separate ticket